easybuild/easyconfigs/t/tqdm/tqdm-4.66.2-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,27 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'PythonPackage'
+
+name = 'tqdm'
+version = '4.66.2'
+
+homepage = "https://github.com/tqdm/tqdm"
+description = """A fast, extensible progress bar for Python and CLI"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['6cd52cdf0fef0e0f543299cfc96fec90d7b8a7e88745f411ec33eb44d5ed3531']
+
+builddependencies = [
+    ('binutils', '2.40'),
```

Comment:
was adding hatchling in 4.66.1 for GCCcore 12.3.0 a mistake?

Reply:
I guess it was necessary at the time, but not anymore thanks to #19777

We should probably revisit easyconfigs that have `hatchling` as a build dependency and clean them up, right @lexming?

Reply:
#20200